pyface/ui/wx/progress_dialog.py

```diff
@@ -46,20 +47,39 @@ def __init__(
         maximum=100,
         direction="horizontal",
         size=(200, -1),
+        **traits,
     ):
         """
         Constructs a progress bar which can be put into a panel, or optionaly,
         its own window
 
         """
-        self._max = maximum
-        self.parent = parent
+        create = traits.pop("create", True)
+
+        # XXX minimum is ignored - it either should be deprecated or supported
```

Comment:
Can we have an issue please - to remove it outright for the next major release? I don't think we have the resources to figure out how to support `minimum` at the moment - and we can implement that functionality from scratch if we ever need to in the future.